src/opm/material/fluidmatrixinteractions/EclMaterialLawManager.cpp

```diff
@@ -140,75 +140,78 @@ initParamsForElements(const EclipseState& eclState, size_t numCompressedElems,
     initParams.run(fieldPropIntOnLeafAssigner, lookupIdxOnLevelZeroAssigner);
 }
 
+// TODO: Better (proper?) handling of mixed wettability systems - see ecl kw OPTIONS switch 74
+// Note: Without OPTIONS[74] the negative part of the Pcow curve is not scaled
 template<class TraitsT>
 std::pair<typename TraitsT::Scalar, bool> EclMaterialLawManager<TraitsT>::
 applySwatinit(unsigned elemIdx,
               Scalar pcow,
               Scalar Sw)
 {
+    // Default is no SWATINIT scaling of the negative part of the Pcow curve, so look up saturation using the input Pcow curve
+    if (pcow <= 0.0) {
+        return {Sw, /*newSwatInit*/ true};
+    }
```

Comment:
If I understand the PR. The only significant change is that you return true instead of false here to indicate that you compute swatinit from the pcow curve if pcow <= 0.